uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -274,6 +273,19 @@ define([
       LeavePopup.openModal(leaveRequest, leaveType, selectedContactId);
     }
 
+    /**
+     * Opens a leave popup for a specific action
+     *
+     * @param {String} action
+     */
+    function openLeavePopupForAction (action) {
+      LeavePopup.openModal(vm.leaveRequest,
+        vm.leaveRequest.request_type,
+        vm.leaveRequest.contact_id,
+        true,
```

Comment:
Is it always true? is there a place where it is false?
if not we dont need to send the param.

Reply:
No, it is not always true. This param forces the balance recalculation on modal open. It is used, yes, in the Leave Popup Service:

```js
function openModalByID (leaveRequestID) {
  return LeaveRequest.find(leaveRequestID)
    .then(function (leaveRequest) {
      return checkPermissionBeforeOpeningPopup(leaveRequest)
        .then(function (hasPermission) {
          if (hasPermission) {
            openModal(leaveRequest, leaveRequest.request_type, leaveRequest.contact_id);
          } else {
            notification.error('Error', 'You dont have permission to see this leave request');
          }
        });
    })
    .catch(function (errorMsg) {
      notification.error('Error', errorMsg);
    });
}
```